src/components/publicApi/ChangePasswordScreen.tsx

```diff
@@ -2,25 +2,28 @@ import { EdgeAccount, EdgeContext } from 'edge-core-js'
 import * as React from 'react'
 
 import { useClearOnUnmount } from '../../hooks/useClearOnUnmount'
+import { OnLogEvent } from '../../types/ReduxTypes'
 import { Router } from '../navigation/Router'
 import { ReduxStore } from '../services/ReduxStore'
 
 interface Props {
   account: EdgeAccount
   context: EdgeContext
   onComplete: () => void
+  onLogEvent: OnLogEvent
```

Comment:
These need to be optional otherwise we have a breaking change to the login-ui API